src/index.ts

```diff
@@ -53,6 +53,7 @@ export const SYLVESTER_ADDRESS = getContractAddressForDeployment({
 export const SYLVESTER_POLYGON_ADDRESS = getContractAddressForDeployment({
```

Comment:
Let's remove all of these exported addresses!

Reply:
done

Reply:
https://github.com/re-nft/sdk/pull/9#discussion_r1121742555